src/styles/createGenerateClassName.js

```diff
@@ -9,15 +9,20 @@ let generatorCounter = 0;
 // It's inspired by
 // https://github.com/cssinjs/jss/blob/4e6a05dd3f7b6572fdd3ab216861d9e446c20331/src/utils/createGenerateClassName.js
 export default function createGenerateClassName(options = {}) {
-  const { dangerouslyUseGlobalCSS = false } = options;
+  const { dangerouslyUseGlobalCSS = false, productionPrefix = 'jss' } = options;
   const escapeRegex = /([[\].#*$><+~=|^:(),"'`\s])/g;
   let ruleCounter = 0;
 
-  // HMR can lead us to instantiating many class name generator.
+  // - HMR can lead us to instantiating many class name generator.
   // The warning is only triggered in production.
-  // We expect people from instantiating a class name generator per new request on the server.
+  // - We expect people from instantiating a class name generator per new request on the server.
   // The warning is only triggered client side.
```

Comment:
"We expect a class name generator to be instantiated per new request on the server, so the warning is only triggered client side."